### ISSUE TITLE
First Web API: .NET 9 update for OpenAPI

### ISSUE THREAD

**Issue by BradleySharp** [dotnet/AspNetCore.Docs#34148]
### Description

The step:
Visual Studio launches the default browser and navigates to https://localhost:<port>/swagger/index.html

Suggests you'll get a SwaggerUI page but with dotnet 9 openapi you won't.

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-9.0&tabs=visual-studio

Additional notes by author:
This topics has not been updated yet to 9.0. The doc and app sample/code for version 9 should reflect the latest template, (such as changes in Program.cs), updated to use openAPI correctly for .NET 9, screenshots updated, and target .NET version step updated.

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/first-web-api.md

### Document ID

95f2549e-29f9-e31a-b4d0-432b971643ff

### Article author

@wadepickett

[Related Issues](https://github.com/dotnet/AspNetCore.Docs/issues?q=is%3Aissue+is%3Aopen+95f2549e-29f9-e31a-b4d0-432b971643ff)

---
[Associated WorkItem - 350920](https://dev.azure.com/msft-skilling/Content/_workitems/edit/350920)

**Comment by wadepickett** [comment]
@BradleySharp, you are correct, thanks for pointing this out!  This one was missed.  I will get it updated in a next sprint.

**Comment by wadepickett** [comment]
Closing as dupe to #33001 which is already under way this current sprint.
